Tests/LibCrypto/TestBigInteger.cpp

```diff
@@ -10,6 +10,7 @@
 #include <LibCrypto/NumberTheory/ModularFunctions.h>
 #include <LibTest/TestCase.h>
 #include <cstring>
```

Comment:
(not a problem with your patch but...) wait what's this doing here?

Reply:
Removed it in the commit adding the math include